Lesson-11/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"liveServer.settings.port": 5501
+}
```

Comment:
why gitignore is empty? where is node_modules?
![image](https://user-images.githubusercontent.com/20189313/141701377-25cccbbc-7109-460d-a30d-6aac2eb5b441.png)